cockpit/cats-plugin/cats.js

```diff
@@ -14,19 +14,19 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.camunda.bpm.cockpit.plugin.failedjobs.resources;
 
-import javax.ws.rs.Path;
-//import javax.ws.rs.PathParam;
+export default {
+  id: "Cats Plugin",
+  pluginPoint: "cockpit.dashboard",
+  priority: 5,
+  render: container => {
+    const header = document.createElement("h1");
+    header.innerHTML = "<h1>Cats!</h1>";
```

Comment:
It's already an h1 element
```suggestion
    header.innerHTML = "Cats!";
```

Reply:
don't know why those 2 are not displayed next to eachother: https://github.com/camunda/camunda-bpm-examples/pull/45#discussion_r466879380

Reply:
Decided to go for https://github.com/camunda/camunda-bpm-examples/pull/45#discussion_r466879380